Ошибка: HTTPConnectionPool(host='99', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000002027AB1BBE0>: Failed to resolve '99' ([Errno 11001] getaddrinfo failed)"))